ui/static/js/app.js

```diff
@@ -612,3 +612,8 @@ function showToast(label, iconElement) {
         }
     }
 }
+
+/** Navigate to the new subscription page. */
+function goToAddSubscription() {
+    window.location.href = '/subscribe';
```

Comment:
The URL should not be hardcoded like this because it will not work for people that use a custom subdirectory (example: `https://example.org/miniflux/subscribe`).

Existing examples: https://github.com/miniflux/v2/blob/master/template/templates/common/layout.html#L50-L51